qcodes/instrument_drivers/Lakeshore/Model_325.py

```diff
@@ -1,11 +1,87 @@
 import numpy as np
-from typing import cast
+from typing import cast, Union, List, Tuple, Iterable
+from itertools import takewhile, repeat
+import os
 
 from qcodes import VisaInstrument, InstrumentChannel, ChannelList
 from qcodes.utils.validators import Enum, Numbers
 from qcodes.instrument.group_parameter import GroupParameter, Group
 
 
+def read_curve_file(file_path: str) -> dict:
+    """
+    Read a curve file with extension *.330
+    The format of the file is as follows:
+
+    Item1: value
+    Item2: value
+    Item3: value
+
+    Header1   Header2   Header3
+    data11     data12    data13
+    data21     data22    data23
```

Comment:
please also document the format of the returned dictionary.

Reply:
Ok